src/other/feedback/feedback-en.hbs

```diff
@@ -179,7 +179,12 @@
 			</section>
 		</section>
 
-		<input type="submit" name="fbsbmt" id="fbsbmt" value="Submit feedback" class="btn btn-primary" /> <input type="reset" value="Reset" class="btn btn-default" />
+		<div class="form-group">
+			<input type="submit" name="fbsbmt" id="fbsbmt" value="Submit feedback" class="btn btn-primary" />
+		</div>
+		<div class="form-group mrgn-tp-md">
+			<input type="reset" value="Reset page to defaults" class="btn btn-link" style="padding: 0;"/>
```

Comment:
We should avoid inline style in our working example

Reply:
A custom class would need to be made.